lib/src/connect_config.dart

```diff
@@ -863,12 +863,13 @@ Future<void> parseDSNIntoConfig(
 
 Future<void> parseCloudInstanceNameIntoConfig(ResolvedConnectConfig config,
     SourcedValue<String> cloudInstanceName, String? stashPath) async {
-  final instanceParts = cloudInstanceName.value.split('/');
+  final normalisedInstanceName = cloudInstanceName.value.toLowerCase();
+  final instanceParts = normalisedInstanceName.split('/');
```

Comment:
Are we missing `.value` calls here and on line 899?

Reply:
Oh, I guess `.value` is something from `SourcedValue`?

Reply:
No, that's only needed for `cloudInstanceName` because it has a type of `SourcedValue<String>`, which is just a wrapper to track the source of the connection param along with the value: https://github.com/edgedb/edgedb-dart/blob/main/lib/src/connect_config.dart#L123-L132